pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
```

Comment:
note: i have updated humanloop-docs with the pytest-xdist dependency; we'll regenerate the sdk and rebase this PR on top of it before merging